### ISSUE TITLE
Implement Tie-game checks

### ISSUE THREAD

**Issue by ry-prz** [tyheise/tictactoe#1]
This feature should:

1. Have game logic that identifies when the game has come to a tie (i.e., there is no longer a chance for either player X or O to win)

2. After a move where the game logic identifies a tie, display a message that will notify the players that the game is in a tie, and end the game.

**One thing we need to clarify about the first part though.** Should the game end:
 a. On the first move that causes a tie (i.e. before all spaces are filled), or
 b. After all spaces have been filled in a tie-game?

**Comment by tyheise** [comment]
I think the game should end after all spaces have been filled. 

Implementation will be much easier without any large downsides for the user.

**Comment by ry-prz** [comment]
Yeah I agree. Also, option B matches up with every other tictactoe app I have used.

It could just be me, but the wording for this feature on the assignment document makes it sound like it's wanting option A. Maybe you guys can ask the TA during the lab to clarify?

Regardless, I'll be implementing B for now since it is relatively simple.
